source/includes/gcp/_k8_releases.md

```diff
@@ -143,5 +143,175 @@ Attributes | &nbsp;
 
+### Get release
 
+```shell
+curl -X GET \
+   -H "MC-Api-Key: your_api_key" \
+   "https://cloudmc_endpoint/v1/services/k8s/anenvironment/releases/pspensieri/aerospike-1579797954?cluster_id=projects/cmc-k8s-enabled-llb/locations/us-central1-a/clusters/standard-cluster-1"
+
+# The above command returns JSON structured like this:
+```
+```json
+{
+  "data": {
+    "id": "pspensieri/aerospike-1579797954",
+    "name": "aerospike-1579797954",
+    "info": {
+      "first_deployed": "2020-01-23T11:45:57.255189-05:00",
+      "last_deployed": "2020-01-24T19:06:12.956425-05:00",
+      "deleted": "",
+      "description": "Rollback to 2",
+      "status": "installed",
+      "notes": "The Aerospike can be accessed via port 3000 on the following DNS name from within your cluster:\n\n  aerospike-1579797954.pspensieri.svc.cluster.local\n\nYou can connect to aeropike in your local machine using port-forwarding:\n\n  export POD_NAME=$(kubectl get pods --namespace pspensieri -l \"app=aerospike,release=aerospike-1579797954\" -o jsonpath=\"{.items[0].metadata.name}\")\n  kubectl  --namespace pspensieri port-forward $POD_NAME 3000:3000\n"
+    },
+    "chart": {
+      "metadata": {
+        "name": "aerospike",
+        "home": "http://aerospike.com",
+        "sources": [
+          "https://github.com/aerospike/aerospike-server"
+        ],
+        "version": "0.3.2",
+        "description": "A Helm chart for Aerospike in Kubernetes",
+        "keywords": [
+          "aerospike",
+          "big-data"
+        ],
+        "maintainers": [
+          {
+            "name": "kavehmz",
+            "email": "kavehmz@gmail.com"
+          },
+          {
+            "name": "okgolove",
+            "email": "okgolove@markeloff.net"
+          }
+        ],
+        "icon": "https://s3-us-west-1.amazonaws.com/aerospike-fd/wp-content/uploads/2016/06/Aerospike_square_logo.png",
+        "apiVersion": "v1",
+        "appVersion": "v4.5.0.5",
+        "deprecated": false
+      },
+      "templates": [
+        {
+          "name": "templates/NOTES.txt",
+          "data": "VGhlIEFlcm9zcGlrZSBjYW4gYmUgYWNjZXNzZWQgdmlhIHBvcnQgMzAwMCBvbiB0aGUgZm9sbG93aW5nIEROUyBuYW1lIGZyb20gd2l0aGluIHlvdXIgY2x1c3RlcjoKCiAge3sgdGVtcGxhdGUgImFlcm9zcGlrZS5mdWxsbmFtZSIgLiB9fS57eyAuUmVsZWFzZS5OYW1lc3BhY2UgfX0uc3ZjLmNsdXN0ZXIubG9jYWwKCllvdSBjYW4gY29ubmVjdCB0byBhZXJvcGlrZSBpbiB5b3VyIGxvY2FsIG1hY2hpbmUgdXNpbmcgcG9ydC1mb3J3YXJkaW5nOgoKICBleHBvcnQgUE9EX05BTUU9JChrdWJlY3RsIGdldCBwb2RzIC0tbmFtZXNwYWNlIHt7IC5SZWxlYXNlLk5hbWVzcGFjZSB9fSAtbCAiYXBwPXt7IHRlbXBsYXRlICJhZXJvc3Bpa2UubmFtZSIgLiB9fSxyZWxlYXNlPXt7IC5SZWxlYXNlLk5hbWUgfX0iIC1vIGpzb25wYXRoPSJ7Lml0ZW1zWzBdLm1ldGFkYXRhLm5hbWV9IikKICBrdWJlY3RsICAtLW5hbWVzcGFjZSB7eyAuUmVsZWFzZS5OYW1lc3BhY2UgfX0gcG9ydC1mb3J3YXJkICRQT0RfTkFNRSAzMDAwOjMwMDAK"
```

Comment:
Not sure if we want to include this base64 encoded data. Thoughts?

Reply:
trunc them a bit to make shorter and invalid. We don't want to have them release decode